config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Is it related to MongoDB express? It is better to make a folder specific to this. 
For example, MongoDB-express/config . 

Kindly check if it makes sense.